CMakeLists.txt

```diff
@@ -1,23 +1,38 @@
 # Copyright The OpenTelemetry Authors
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.9)
+cmake_minimum_required(VERSION 3.10)
 
-# See https://cmake.org/cmake/help/v3.3/policy/CMP0057.html required by certain
-# versions of gtest
+# These policies belowed are used to keep a more strick behavior of cmake
+cmake_policy(SET CMP0054 NEW)
```

Comment:
Can we add comment on what these policies do, though I understand some existing policies are missing the comment :)

Reply:
These old policies are removed now.